site/en/guide/ragged_tensor.ipynb

```diff
@@ -1389,7 +1338,7 @@
       },
       "outputs": [],
       "source": [
-        "print(rt[1])                        # Second row (2-D RaggedTensor)"
+        "print(rt[1])                        # Second row (2D RaggedTensor)"
```

Comment:
2-D. Similarly, for other instances below.

Reply:
I'd keep the changes, since the "ND"—such as "2D" or "3D"—appear to be widely used. For instance, https://github.com/tensorflow/docs/blob/master/site/en/tutorials/generative/cvae.ipynb and https://github.com/tensorflow/docs/blob/master/site/en/guide/tensor.ipynb

Reply:
@lamberta @MarkDaoust @anirudh161 If the convention is "N-D" for N dimensions and not "ND", we can lint the docs accordingly. LMKWYT 👍

Reply:
Yes, the convention is "N-D".
2D and 3D are common but higher dimensions are unfamiliar. I'd be open to adding this as a lint check